doc/install.txt

```diff
@@ -412,7 +412,7 @@ that we use.
 3) Install the ATLAS library. ATLAS is an open source optimized version of
 BLAS. You can install a precompiled version on most OSes, but if you're willing
 to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especialy on more recent computers, against the
+speed-ups of up to 3×, especially on more recent computers, against the
```

Comment:
The accent in NEWS.txt is a separate thing then this PR. When we make a new
release we can see that again.

Martin, can you do that small change?

On Mon, Jan 12, 2015 at 1:59 PM, abergeron <notifications@github.com> wrote:

> In doc/install.txt
> <https://github.com/Theano/Theano/pull/2391#discussion-diff-22812334>:
>
> > @@ -412,7 +412,7 @@ that we use.
> >  3) Install the ATLAS library. ATLAS is an open source optimized version of
> >  BLAS. You can install a precompiled version on most OSes, but if you're willing
> >  to invest the time, you can compile it to have a faster version (we have seen
> > -speed-ups of up to 3x, especialy on more recent computers, against the
> > +speed-ups of up to 3×, especially on more recent computers, against the
>
> In this case I don't think the slight visual improvement is worth it.
> There is always the risk that browsers (especially on windows) will not
> have a proper Unicode font and will display a box instead of an x.
>
> Also what kind of problems do you encounter with accents in NEWS.txt?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2391/files#r22812334>.
>